net.danielgm.CloudCompare.yaml

```diff
@@ -0,0 +1,290 @@
+app-id: net.danielgm.CloudCompare
+runtime: org.kde.Platform
+runtime-version: "5.12"
+sdk: org.kde.Sdk
+command: CloudCompare
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --share=network
+  - --filesystem=home
+  - --filesystem=host
+
+cleanup:
+    - /bin/applygeo
+    - /bin/cgal_*
+    - /bin/CreateDOMDocument
+    - /bin/DOM*
+    - /bin/EnumVal
+    - /bin/flann*
+    - /bin/gdal*
+    - /bin/geotifcp
+    - /bin/gnmanalyse
+    - /bin/gnmmanage
+    - /bin/listgeo
+    - /bin/MemParse
+    - /bin/nearblack
+    - /bin/ogr*
+    - /bin/pdal*
+    - /bin/PParse
+    - /bin/PSVIWriter
+    - /bin/Redirect
+    - /bin/SAX*
+    - /bin/SCMPrint
+    - /bin/SEnumVal
+    - /bin/StdInParse
+    - /bin/testepsg
+    - /bin/XInclude
+
+    - /lib/cmake
+    - /lib/debug
+    - /lib/gdalplugins
+    - /lib/pkgconfig
+    - "*.a"
+    - "*.la"
+
+    - /share/doc
+    - /share/eigen3
+    - /share/flann
+    - /share/gdal
+    - /share/info
+    - /share/man
+    - /share/pcl-1.9
+    - /share/pkgconfig
+    - /share/proj
+
+    - /include
+
+modules:
+  - name: MPFR
+    sources:
+      - type: archive
+        url: https://www.mpfr.org/mpfr-current/mpfr-4.0.2.tar.xz
+        sha256: 1d3be708604eae0e42d578ba93b390c2a145f17743a744d8f3f8c2ad5855a38a
+    buildsystem: autotools
+    builddir: true
```

Comment:
```suggestion

```

Reply:
Is `builddir` only used for `cmake` projects? Or why should I remove this?

Reply:
Is it already set by default by flatpak-builder. You can remove that from every module